folium/folium.py

```diff
@@ -404,9 +413,12 @@ def div_markers(self, locations=None, popups=None, marker_size=10, popup_width=3
                                        'icon': "{'icon':"+icon_name+"}"
                                        })
 
-            popup_out = self._popup_render(popup=popup, mk_name='div_marker_{0}_'.format(call_cnt),
```

Comment:
Yep.  Much better.  I am just going to call it `mk_name` instead of `mk_out` to be more consist.